src/pages/Main/Main.tsx

```diff
@@ -77,18 +77,16 @@ const Main: React.FC = () => {
   return (
     <div className={"MainPage"}>
       <Input onChange={onChangeCardType}/>
-      <SupportCardListContainer cardType={state.cardType} onClickItem={addCard} />
+      <SupportCardListContainer cardType={state.cardType} onClickItem={addCard} selectedList={state.uuids} />
```

Comment:
```suggestion
      <SupportCardListContainer
        cardType={state.cardType}
        onClickItem={addCard}
        selectedList={state.uuids} />
```
이것도 취향이지만... 이 코드베이스에서는 라인이 80줄이 넘으면 모든 parameter를 줄바꿈 합니다

Reply:
이와 관련해서 별도로 사용하시는 포매터가 있으신가요?
일단 개발 환경 에디터에서는 저도 maxColumn을 80으로 변경해놓겠습니다

Reply:
별도로 사용하는건 아직 없습니다 ㅋㅋ 나중에 린터라도 달아야겠군요